docs/tables/azuread_domain.md

```diff
@@ -0,0 +1,28 @@
+# Table: azuread_domain
+
+Azure Active Directory (Azure AD) lets you use domains to manage access to your cloud-based resources.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  id,
+  is_admin_managed,
+  is_verified,
+  supportedServices
```

Comment:
This column name looks weird - is the column name incorrect in the table (should be `supportedServices`), or is the example wrong?